views/movies/upcoming.php

```diff
@@ -0,0 +1,13 @@
+
+<script src="../../models/movies/upcoming.js"></script>
+
+<main class="container" id="infscroll">
+<section class="movies" id="movies">
+    <h2>Upcoming Movies</h2><br>
+	<div class="row" id="render">
+    <center id='loading'> <p>Loading movies..</p> </center> 
```

Comment:
Missing indentation here and the line below.

Reply:
Noted.